doc/Installation/Installation-(RHEL-CentOS).md

```diff
@@ -155,12 +160,26 @@ Next, add the following to `/etc/httpd/conf.d/librenms.conf`
 ```
 
 __Notes:__
-If you are running Apache 2.2.18 or higher then change `AllowEncodedSlashes` to `NoDecode` and append `Require all granted` underneath `Options FollowSymLinks MultiViews`.
+If you are running Apache 2.2.18 or higher (true if centos 7) you will want to change `AllowEncodedSlashes` to `NoDecode` and append `Require all granted` underneath `Options FollowSymLinks MultiViews`, as shown below:
```

Comment:
This is definitely better but maybe it's worth just having both examples with the headers of the versions they work with and say pick one or the other depending on apache version.